theme_paptic/views/customizations.xml

```diff
@@ -303,7 +303,7 @@
     </xpath>
     <!-- Shape -->
     <xpath expr="//div[hasclass('container')]" position="before">
-        <div class="o_we_shape o_web_editor_Airy_12_001 o_we_flip_y"/>
```

Comment:
EDIT: I guess that's because this just was forward-ported now? https://github.com/odoo/odoo/pull/117652

Reply:
> EDIT: I guess that's because this just was forward-ported now? [odoo/odoo#117652](https://github.com/odoo/odoo/pull/117652)

This shouldn't be related to the RGBA color fix because it doesn't use RGBA colors...
Anyway I'm not able the reproduce the bug, could you retry?

Reply:
Strange, don't have the bug anymore :shrug: